src/components/Card.jsx

```diff
@@ -0,0 +1,66 @@
+import React, { useState, useEffect } from 'react';
+import PropTypes from 'prop-types';
+
+const isInitialCard = (card) => {
+  return Object.values(card).some((value) => value !== '');
+};
+
+function Card({ card }) {
+  const [isEmptyCard, setIsEmptyCard] = useState(true);
+
+  useEffect(() => {
+    return isInitialCard(card) ? setIsEmptyCard(false) : setIsEmptyCard(true);
+  }, [card]);
```

Comment:
별도의 상태가 필요한 건지도 궁금하네요!
```suggestion
  const isEmptyCard = Object.values(card).some((value) => value !== '')
```

Reply:
card 상태가 변하면 render가 되니깐 이렇게 하면 깔끔하군요! 감사합니다 :)
아래와 같이 변경하였습니다 ㅎ
https://github.com/woowacourse/react-payments/pull/106/commits/72b8b1174e0e2b1d4ba5e61e6a3437613df34431